Sources/MongoSwiftSync/ChangeStream.swift

```diff
@@ -4,47 +4,64 @@ import MongoSwift
 /// - SeeAlso: https://docs.mongodb.com/manual/changeStreams/
 public class ChangeStream<T: Codable>: Sequence, IteratorProtocol {
     /// A `ResumeToken` associated with the most recent event seen by the change stream.
-    public var resumeToken: ResumeToken? { fatalError("unimplemented") }
+    public var resumeToken: ResumeToken? {
+        return self.asyncChangeStream.resumeToken
+    }
+
+    /// Indicates whether this change stream has the potential to return more data.
+    public var isAlive: Bool {
+        return self.asyncChangeStream.isAlive
+    }
 
-    /// The error that occurred while iterating the change stream, if one exists. This should be used to check
-    /// for errors after `next()` returns `nil`.
-    public var error: Error? { fatalError("unimplemented") }
+    private let asyncChangeStream: MongoSwift.ChangeStream<T>
+
+    /// The client this change strem descended from.
+    private let client: MongoClient
```

Comment:
The sync change stream needs a reference to the sync client, otherwise it will go out of scope and transitively close the async client before this sync change stream goes out of scope. As a result, I had to go back and add the references to the sync client to the sync db and collection.